.github/workflows/ci.yml

```diff
@@ -184,7 +190,7 @@ jobs:
           path: |
             ./target
           key:
-            d-${{ matrix.os }}-${{ matrix.kind }}-${{ hashFiles('Cargo.lock') }}-${{ hashFiles('Cargo.lock') }}
+            a-cargo-target-${{ matrix.os }}-${{ matrix.profile }}-${{ hashFiles('Cargo.lock') }}
```

Comment:
The purpose of this patch is to reduce the number of distinct cargo-target caches we store by keying on build profile rather than "matrix.kind". The worry is that we're hitting our 5G cache limit before we're even able to store one set of caches, so we are continually rolling.